tests/fuzzer.c

```diff
@@ -366,6 +370,18 @@ static int basicUnitTests(U32 const seed, double compressibility)
     }
     DISPLAYLEVEL(3, "OK \n");
 
+    /* Need to write some better tests for this */
+    DISPLAYLEVEL(3, "test%3u : ZSTDMT_depThreadPool dep test : ", testNb++);
+    {
+      int data = 0;
+      ZSTDMT_DepThreadPoolCtx* ctx = ZSTDMT_depThreadPool_createCtx(2, 2);
+      size_t jobId = ZSTDMT_depThreadPool_addJob(ctx, FUZ_depThreadPool_callbackFn1, &data, 0, NULL);
+      size_t deps[1] = {jobId};
+      jobId = ZSTDMT_depThreadPool_addJob(ctx, FUZ_depThreadPool_callbackFn2, &data, 1, deps);
```

Comment:
Both jobs write to the same `data`.
The only protection is that the second job must wait for the first one to be completed before starting,
thus solving risks of race conditions.
It basically tests that the dependency is implemented correctly.

I recommend to explain the logic of this test in code comment.

Reply:
Yes. Better tests to come. This is just a place holder:)